k4GaudiPandora/include/DDPandoraPFANewAlgorithm.h

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright (c) 2020-2024 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *  @file   DDMarlinPandora/include/DDPandoraPFANewAlgorithm.h
+ *
+ *  @brief  Header file for the pandora pfa new algorithm class.
+ *
+ *  $Log: $
+ */
+
+#ifndef DDPANDORAPFANEWALGORITHM_H
+#define DDPANDORAPFANEWALGORITHM_H 1
+
+#include "DDCaloHitCreator.h"
+#include "DDGeometryCreator.h"
+#include "DDMCParticleCreator.h"
+#include "DDPfoCreator.h"
+#include "DDTrackCreatorBase.h"
+
+// k4FWCore
+#include <k4FWCore/DataHandle.h>
+#include <k4FWCore/BaseClass.h>
+#include <k4FWCore/Transformer.h>
+#include "k4Interface/IGeoSvc.h"
+#include <Gaudi/Property.h>
+#include "GaudiKernel/MsgStream.h"
+
+#include "DD4hep/DetType.h"
+#include "DD4hep/DetectorSelector.h"
+
+namespace pandora {
+  class Pandora;
+}
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+/**
+ *  @brief  DDPandoraPFANewAlgorithm class
+ */
+struct DDPandoraPFANewAlgorithm final : 
```

Comment:
```suggestion
struct DDPandoraPFAAlgorithm final : 
```
I think there is no *old* algorithm yet, so there is also no reason to name this *new* algorithm.